src/hermes/commands/deposit/invenio.py

```diff
@@ -438,3 +446,34 @@ def _get_license_identifier(ctx: CodeMetaContext, license_api_url: str):
     response.raise_for_status()
 
     return response.json()["id"]
+
+
+def _get_community_identifiers(ctx: CodeMetaContext, communities_api_url: str):
+    """Get Invenio community identifiers from config.
+
+    This function gets the communities to be used for the deposition on an Invenio-based
+    site from the config and checks their validity against the site's API. If one of the
+    identifiers can not be found on the site, a :class:`MisconfigurationError` is
+    raised.
+    """
+
+    communities = config.get("deposit").get("invenio", {}).get("communities")
+    if communities is None:
+        return None
+
+    session = requests.Session()
+    session.headers = {"User-Agent": hermes_user_agent}
```

Comment:
**question:** I like the idea of a custom user agent... maybe we should move the network handling code to a common submodule (however, out-of-scope for poc).

Reply:
Agreed. This is the second time this has come up so I'll write an issue for this.

Reply:
https://github.com/hermes-hmc/workflow/issues/165